tests/handler_tests/test_qdrant_handler.py

```diff
@@ -0,0 +1,445 @@
+from unittest.mock import patch
+
+import pytest
+import pandas as pd
+from mindsdb_sql import parse_sql
+
+from unit.executor_test_base import BaseExecutorTest
```

Comment:
should be` from unit.executor_test_base import BaseExecutorTest` as tests are executed from package root

Reply:
https://github.com/mindsdb/mindsdb/pull/7833/commits/183214419563b7259b8c8c263d5cd4e7c535d3b3